### ISSUE TITLE
about training information

### ISSUE THREAD

**Issue by Nomination-NRB** [chendatouha/dt_tryon#3]
I‘m greatful for the training code you provided, but I also have some question: 
 
 What is the minimum GPU requirement for training?

Besides, I wonder what the normal loss curve looks like. And How many epochs should I set

**Comment by chendatouha** [comment]
At least 2 GPUs with 40GB of VRAM each are required to train in zero2 mode using the deepspeed framework. I'm sorry, I didn't save the loss curve. Based on your actual dataset, I think we should train for at least 10,000 steps.

**Comment by Nomination-NRB** [comment]
> At least 2 GPUs with 40GB of VRAM each are required to train in zero2 mode using the deepspeed framework. I'm sorry, I didn't save the loss curve. Based on your actual dataset, I think we should train for at least 10,000 steps.

OK，thanks a lot
